gyp/platform-ios.gypi

```diff
@@ -61,6 +61,7 @@
         '../platform/ios/NSString+MGLAdditions.m',
         '../platform/ios/vendor/SMCalloutView/SMCalloutView.h',
         '../platform/ios/vendor/SMCalloutView/SMCalloutView.m',
+        '../platform/ios/resources/',
```

Comment:
So that it's visible in the project — it's not needed for compilation. Had a situation last week where I wanted to show Brad the compass images but needed to jump out to Github to find them.